### DISCUSSION TITLE
thanhle.blog/don-bay

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#94] [Ideas]
# thanhle.blog/don-bay

https://thanhle.blog/blog/don-bay

**Comment by thanhlmm** [comment]
Mình có được TopDev tặng 10 vé tham gia Vietnam Mobile Submit. Bạn bè ai ai muốn đi thì sử dụng nhé

Truy cập: https://topdev.vn/vms/tickets
Nhập mã: LTHANH@VMS2024 (Mã code được sử dụng tối đa 10 lần, chỉ áp dụng cho vé tiêu chuẩn)

**Comment by hai123vn** [comment]
Một bài viết thật là bổ X, cảm ơn anh. Chúc anh 1 ngày thật nhiều năng lượng future đến đâu thắng đến đó :v

**Reply by thanhlmm** [reply]
cảm ơn em

**Comment by anhdungadg** [comment]
Dùng đòn bẩy đẩy ví von so sánh hai mặt với ảo tưởng hay và truyền cảm hứng quá, cảm ơn tác giả 👏👏

**Comment by hai123vn** [comment]
Anh ơi, khi nào anh ra thêm bài nữa vậy ạ 😅

**Reply by thanhlmm** [reply]
:))) cuối tuần

**Comment by quan-nh2** [comment]
Tự thấy mình ở đâu đây, cám ơn anh đã chia sẻ.
